.gitignore

```diff
@@ -60,6 +60,15 @@ node_modules
 
 *.db
 
-/stash
 dist
-.DS_Store
\ No newline at end of file
+.DS_Store
+
+####
+# stash
+####
+/stash
+ui/**/package-lock.json
+ui/**/yarn.lock
+config.yml
+icon.png
+stash-go.sqlite
```

Comment:
These are not build artefacts. These are files generated when running stash from the source directory. `yarn.lock` is intended to be in the source tree. I'm not sure what `package-lock.json` is.